pom.xml

```diff
@@ -162,6 +182,7 @@ LGPL.
 					</execution>
 				</executions>
 			</plugin>
+			<!-- akley: will not run in batch mode (maven) 
```

Comment:
Yes, it will not, but the same time if I pull it to master it will not get without signature to maven central. User instead `-Dgpg.skip=true` in your batch.